etcdctl/README.md

```diff
@@ -1083,6 +1083,47 @@ echo ${transferee_id}
 # Leadership transferred from 45ddc0e800e20b93 to c89feb932daef420
 ```
 
+### DOWNGRADE \<subcommand\>
+
+Downgrade provides commands to downgrade cluster version
+
+### DOWNGRADE VALIDATE \<TARGET_VERSION\>
+
+DOWNGRADE VALIDATE validate downgrade capability before starting downgrade
+
+#### Example
+
+```bash
+./etcdctl downgrade validate 3.5.0
+Downgrade validate success, cluster version 3.6.0
```

Comment:
Do we expect automated scripts to parse that output, or there is more suitable format for automation, e.g.: 

```
currentClusterVersion: 3.6.0
proposedClusterVersion: 3.5.0
validationStatus: SUCCESS
``` ?

Reply:
-  shell we document that the request version needs to be 'major' (x.y.0) ?

Reply:
For automation scripts I think we should add support for json output. 

As for versions, I think we should change it to "{Major}.{Minor}" version.

Reply:
Done